packages/wallet/backend/src/auth/service.ts

```diff
@@ -55,6 +55,12 @@ export class AuthService implements IAuthService {
   }
 
   public async authorize(args: AuthorizeArgs): Promise<AuthorizeResult> {
+    if (
+      args.email === this.env.DEFAULT_WALLET_ACCOUNT.email ||
+      args.email === this.env.DEFAULT_BOUTIQUE_ACCOUNT.email
+    )
+      await this.userService.createDefaultAccount()
```

Comment:
dafault users should be created at startup or using as script as @raducristianpopa suggested

with current approach if I register a new account and login there, I won't be able to use the boutique as the default account for it will still not be created